File: 백준/Silver/1764. 듣보잡/듣보잡.py
N, M = map(int, input().split())
nolook_nolisten = []
no_look = set()

for i in range(N):
    no_look.add(input())
for i in range(M):
    no_listen = input()
    if no_listen in no_look:
        nolook_nolisten.append(no_listen)
        no_look.remove(no_listen)

nolook_nolisten.sort()
print(len(nolook_nolisten))
for people in nolook_nolisten:
    print(people)

File: 백준/Silver/1764. 듣보잡/듣보잡.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

const N = input[0].split(" ").map(Number)[0];
const M = input[0].split(" ").map(Number)[1];

let eme = new Set();
let emeqh = [];

for (let i = 1; i < N + M + 1; i++) {
  if (i <= N) {
    eme.add(input[i].trim());
  } else {
    let qh = input[i].trim();
    if (eme.has(qh)) {
      emeqh.push(qh);
    }
  }
}

emeqh.sort();
console.log(emeqh.length);
for (let gg of emeqh) {
  console.log(gg);
}
